doc/source/indexing.rst

```diff
@@ -1790,7 +1773,7 @@ Evaluation order matters
 
 Furthermore, in chained expressions, the order may determine whether a copy is returned or not.
 If an expression will set values on a copy of a slice, then a ``SettingWithCopy``
-exception will be raised (this raise/warn behavior is new starting in 0.13.0)
+exception will be raised.
```

Comment:
this is only a warning by default (and depends on the value of the option for raise/warn)